cartridge.lua

```diff
@@ -686,6 +686,7 @@ local function cfg(opts, box_opts)
 
         local srv_name = httpd.tcp_server:name()
         log.info('Listening HTTP on %s:%s', srv_name.host, srv_name.port)
+        rawset(_G, '__instance_http_port', srv_name.port)
```

Comment:
I guess all cartridge global variables should have "cartridge" prefix.

Also I have several questions:
Why do you need a global variable, is it possible to get all needed info from service_registry?
Currently http has more options than port. There are also host and webui_prefix. I think graphql API should return them as well.